dcbr-api/email_service/apps.py

```diff
@@ -1,15 +1,20 @@
 from django.apps import AppConfig
 from django.db.models.signals import post_migrate
+import logging
+
+LOGGER = logging.getLogger(__name__)
 
 
 def start_reminder_email_task(sender, **kwargs):
-    from .tasks import send_reminder_email, send_queued_mail
+    from .tasks import send_reminder_email, send_queued_mail, update_registration_status
 
     # reminder emails are scheduled to run daily
     send_reminder_email(repeat=60 * 60 * 24, repeat_until=None)
 
     # queued mail is processed every minute
-    send_queued_mail(repeat=60, repeat_until=None)
+    send_queued_mail(repeat=60 * 60 * 24, repeat_until=None)
```

Comment:
Queued mail should be sent more frequently than once a day, otherwise new registrations won't be receiving their notification email potentially until after 24hrs. Any specific reason to change this? I would be okay with making the task run less frequently than every minute, but once a day doesn't seem often enough.

Reply:
that was not intentional on my part .. I'll fix it